docs/operations/updating_the_stack.rst

```diff
@@ -246,6 +246,24 @@ Deploy the corresponding pipeline to ``concourse-v6``:
 
 ----
 
-Cleanup
+5. Cleanup
 =======
 Close all terminal windows when deployment is complete.
+
+
+----
+
+***************************
+Rotating Basic Auth Secrets
+***************************
+
+To update basic auth secrets for COPS, a dev must copy an ``htaccess`` file sourced from AWS SecretsManager and rotate the secret in the swarm with:
```

Comment:
I've put the command to get the secret in Confluence here since it's a bit more private: https://openstax.atlassian.net/wiki/spaces/CE/pages/670760961/COPS+Basic+Auth+Credentials+Management